train_segmentation.py

```diff
@@ -140,8 +141,15 @@ def get_num_samples(samples_path, params):
         else:
             with h5py.File(samples_path.joinpath(f"{i}_samples.hdf5"), "r") as hdf5_file:
                 num_samples[i] = len(hdf5_file['map_img'])
+                if i == 'trn':
```

Comment:
Is that the more efficient way to get the num of sample? the for loop seem to be to much and heavy to compute.

Reply:
I haven't thought much about it. any suggestions?

Reply:
give me the afternoon to check and I come back to you